pymc_experimental/inference/laplace.py

```diff
@@ -181,10 +222,349 @@ def addDataToInferenceData(model, trace, progressbar):
         default_dims=[],
     )
 
-    trace.add_groups(
+    idata.add_groups(
         {"observed_data": observed_data, "constant_data": constant_data},
         coords=coords,
         dims=dims,
     )
 
-    return trace
+    return idata
+
+
+def fit_mvn_to_MAP(
```

Comment:
`fit_mvn_at_MAP`? I mean technically this function just fits a MVN at a `point`, the user doesn't necessarily have to pass the MAP